README.md

```diff
@@ -15,6 +15,27 @@ with any kind of software without having to release its source code.
 Additionally, the example code has been released to the public domain (see the
 [separate license](examples/COPYING) for more information).
 
+Quick Started
+===============
+
+**Prerequisites** for building libgit2:
+
+1. [CMake](https://cmake.org/)
+2. [Python](https://www.python.org) is used by our test framework.
```

Comment:
Maybe specify that the user should make sure its in his path? Dunno if that is default for the Windows installer, but I think I remember there being at least an option for that